zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -31,11 +31,13 @@
 
 #include <stdio.h>           /* fprintf */
 #include <stdlib.h>          /* malloc */
```

Comment:
If no function call `malloc()` nor `free()` directly, it should be possible to remove this `#include`
(well, if `stdlib` is not included either in any other include. But that should be the case too).
It will also help to spot if any invocation of `free()` is forgotten somewhere.